src/toasts/toasts.js

```diff
@@ -1,24 +1,28 @@
 import React from "react";
 import ReactDOM from "react-dom";
+import { Subject } from "rxjs";
+import ActiveToasts from "./active-toasts.component";
+
+export const toastsSubject = new Subject();
```

Comment:
Using a subject as an event emitter - the `showToast` function emits toast objects into the already-rendered ActiveToasts react component